boards/itsybitsy_m0/examples/ssd1306_terminalmode_128x64_i2c.rs

```diff
@@ -97,7 +96,8 @@ fn main() -> ! {
     let i2c_sercom: bsp::I2cSercom = peripherals.SERCOM3;
     let i2c = bsp::i2c_master(
         &mut clocks,
-        KiloHertz(400),
+        400.kHz(),
+        // KiloHertz(400),
```

Comment:
Maybe just get rid of the commented out line?

Reply:
Good catch, fixed!